packages/js-drive/lib/validator/Validator.js

```diff
@@ -36,12 +37,31 @@ class Validator {
     return Validator.DEFAULT_DASH_VOTING_POWER;
   }
 
+  /**
+   * Get network info
+   *
+   * @returns {ValidatorNetworkInfo}
+   */
+  getNetworkInfo() {
+    return this.networkInfo;
+  }
+
+  /**
+   * Set network info
+   *
+   * @param {ValidatorNetworkInfo} networkInfo
+   */
+  setNetworkInfo(networkInfo) {
```

Comment:
This is required property so let's remove setter and accept it in the constructor